ballerina-to-openapi/src/main/java/io/ballerina/openapi/service/mapper/OpenAPIResourceMapper.java

```diff
@@ -55,6 +55,8 @@
 import java.util.Set;
 
 import static io.ballerina.openapi.service.mapper.Constants.DEFAULT;
+import static io.ballerina.openapi.service.mapper.Constants.OAS_CONVERTOR_113;
+import static io.ballerina.openapi.service.mapper.Constants.OAS_CONVERTOR_125;
```

Comment:
Rather than introducing new constants we could use `DiagnosticMessages` enum for this right ? If we use `DiagnosticMessages.getCode()` it will return the diagnostic code which is what we have in this constant field :thinking: